server.js

```diff
@@ -9,10 +9,12 @@ app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 
 const pantryRouter = require('./api/controllers/routes/pantries');
```

Comment:
Doesn't have to be for this check-in but is there a way we can specify paths somewhere in a settings file so it automatically figures it out or can we have a const for part of the route since it doesn't change?

Reply:
I'm not sure about node. I know you can do that in Typescript.

Reply:
After doing a bit of research, it looks like path aliasing is possible with more external dependencies. Since it isn't a huge issue right now, we could probably do without.